contents/blog/validate-b2b-product-market-fit.md

```diff
@@ -0,0 +1,53 @@
+---
+date: 2021-08-30
+title: Here’s why you’re failing to achieve B2B product market fit
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: marcus-hyett
+featuredImage: ../images/blog/validate-b2b-product-market-fit.png
+featuredImageType: full
+---
+
+Validating B2B product market fit is a common problem for PMs at organizations of all sizes. There is no universal standard for validating product market fit, but there are some great heuristics you can use to understand whether you’re on the right track. Achieving product market fit is essential to increasing investments in marketing, growth, and expansion.
+
+## Defining the market
+The first step towards validating product market fit is to define your market. When starting with a new product, it’s not realistic to say “this is mass market” or “our audience is everyone.” The more focused you are, the better your product will fit your target market.
+
+When defining a potential market, break down the audience by common dimensions (e.g. location, industry, size, etc). Use your company's goals or mission to identify the most valuable and feasible market segment that supports long-term user and revenue growth. 
```

Comment:
One of the things I learned recently is - location/industry/size are the _obvious_ things, however, it could equally be more specific characteristics.

Ie for PostHog it's:

Needs:
* Need to control their user data
* Need to excel at product led growth to remain competitive
Haves:
* Have budget and savvy engineers are the decision makers
* Have achieved product market fit
* Have an unsuccessful central analytics function
* Have deployed our open source product successfully

Reply:
Added